src/renderer/components/cluster-manager/cluster-frame-handler.ts

```diff
@@ -113,9 +112,9 @@ export class ClusterFrameHandler {
 
           return undefined;
         },
-        (view) => {
+        (view: LensView) => {
           logger.info(`[LENS-VIEW]: cluster id=${clusterId} should now be visible`);
-          view.frame.style.display = "flex";
+          view.frame.classList.remove("hidden");
```

Comment:
_FYI:_ It's always better to move all styles management if possible to CSS files and just toggle classnames in JS.